env/initialize.sh

```diff
@@ -114,6 +130,7 @@ while getopts ":f" opt; do
   esac
 done
 
+initialize_emscripten
```

Comment:
nit: Unless there is some reason not to, we can maintain alphabetical order here

Reply:
Done, and also added comments for items that actually depend on others' result. (Accidentally, the alphabetical order is correct for them too.)